### PR TITLE
productpage

### PR DESCRIPTION
**Pull request by JaqKent**
gather product data

### CODE REVIEW

File: src/components/productPage/index.tsx [hunk 1]
```diff
@@ -1,10 +1,37 @@
+import { useContext, useState } from "react";
+import { useNavigate } from "react-router-dom";
 import Carousel from "components/Carrousel";
-import { products } from "./constants";
+import { Product } from "interface/Products";
+import { CartContext } from "context/Cart";
 
 import { ReactComponent as Arrow } from "assets/arrowLeft.svg";
 import styles from "./styles.module.scss";
+import paths from "App/paths";
```
Comment (nathan-hyan) [inline src/components/productPage/index.tsx]: Sort imports

File: src/components/productPage/index.tsx [hunk 2]
```diff
@@ -1,10 +1,37 @@
...
+  const handleAddToCart = () => {
+    addProductToCart(product, quantity);
+  };
+
+  const handleGoToCart = () => {
+    navigate(paths.cart);
+    removeModal();
+  };
+
+  const handlegoToStore = () => {
```
Comment (nathan-hyan) [inline src/components/productPage/index.tsx]: Camel case not correct

**Review by nathan-hyan — CHANGES_REQUESTED** [review]
Amazing job, some comments

File: src/components/productPage/index.tsx (hunk 1 above)
Comment (JaqKent) [reply src/components/productPage/index.tsx]: done

File: src/components/productPage/index.tsx (hunk 2 above)
Comment (JaqKent) [reply src/components/productPage/index.tsx]: done

**Pull request merged**